### DISCUSSION TITLE
Unable to map the input quantity value in the data object importer, the field name is not showing.

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#14588] [General]
while unable to map the input quantity value in the data object importer, the field name is not showing.

Does anyone know the solution.

input is 10 kg, and data component type is input quantity value.
my pimcore community version -v10.5.11

**Comment by mattamon** [comment]
Duplicate  https://github.com/pimcore/pimcore/discussions/14665

**Reply by arulselvan008** [reply]
Hi Mattamon,

I am getting no response from this request, so I created the same question in another group; which group is correct? Please suggest, and do you have any suggestions to my issue?
